### PR TITLE
Changes for Issue #155

### PR DESCRIPTION
**Pull request by atreyamaj**
Fix: #155 

I have made changes to `docs/src/contributing/guidelines.md`  so that new/potential contributors have an easier time navigating. Please do let me know if there are any changes to be made!

Please find the screenshot attached:

Screenshot from 2021-02-25 11-19-19

### CODE REVIEW

File: docs/src/contributing/guidelines.md [hunk 1]
```diff
@@ -55,3 +55,13 @@ that you can modify it and submit a pull request on GitHub later. Don't
 hesitate to ask questions in our [gitter channel](https://gitter.im/JuliaEarth/GeoStats.jl)
 if you are not familiar with the Git workflow. We are looking forward
 to your contributions.
+
+## I'm new to open-source. How do I start contributing?
+
+Contributing to an open-source project for the very first time can be a very daunting task. Don't worry, we were all there at some point as well! To make the process easier and more github-beginner-friendly, our friends over at Invenia have written an article about how to start contributing to open-source and overcome the Mental and Technical barriers that come associated with it :
```
Comment (juliohm) [inline docs/src/contributing/guidelines.md:61]:
```suggestion
Contributing to an open-source project for the very first time can be a very daunting task.
To make the process easier and more github-beginner-friendly, the community has written
an article about how to start contributing to open-source and overcome the mental and
technical barriers that come associated with it : 
```

File: docs/src/contributing/guidelines.md [hunk 2]
```diff
@@ -55,3 +55,13 @@ that you can modify it and submit a pull request on GitHub later. Don't
...
+
+## I'm new to open-source. How do I start contributing?
+
+Contributing to an open-source project for the very first time can be a very daunting task. Don't worry, we were all there at some point as well! To make the process easier and more github-beginner-friendly, our friends over at Invenia have written an article about how to start contributing to open-source and overcome the Mental and Technical barriers that come associated with it : 
+
+https://invenia.github.io/blog/2021/01/29/contribute-open-source
+
+The article will also take you through the steps required to make your first contribution in detail.
+
+
```
Comment (juliohm) [inline docs/src/contributing/guidelines.md:67]:
```suggestion
The article will also take you through the steps required to make your first contribution using
the GitHub pull request workflow:

https://github.com/firstcontributions/first-contributions
```

**Review by juliohm — CHANGES_REQUESTED** [review]
Made a few changes. Please activate the "enable edit from maintainer options" in future PRs so that I can commit small fixes directly and merge.

File: docs/src/contributing/guidelines.md [hunk 3]
```diff
@@ -55,3 +55,15 @@ that you can modify it and submit a pull request on GitHub later. Don't
 hesitate to ask questions in our [gitter channel](https://gitter.im/JuliaEarth/GeoStats.jl)
 if you are not familiar with the Git workflow. We are looking forward
 to your contributions.
+
+## I'm new to open-source. How do I start contributing?
+
+Contributing to an open-source project for the very first time can be a very daunting task.
+To make the process easier and more github-beginner-friendly, the community has written
```
Comment (juliohm) [inline docs/src/contributing/guidelines.md:62]:
```suggestion
To make the process easier and more GitHub-beginner-friendly, the community has written
```

File: docs/src/contributing/guidelines.md [hunk 4]
```diff
@@ -55,3 +55,14 @@ that you can modify it and submit a pull request on GitHub later. Don't
 hesitate to ask questions in our [gitter channel](https://gitter.im/JuliaEarth/GeoStats.jl)
 if you are not familiar with the Git workflow. We are looking forward
 to your contributions.
+
+## I'm new to open-source. How do I start contributing?
+
+Contributing to an open-source project for the very first time can be a very daunting task.
+To make the process easier and more GitHub-beginner-friendly, the community has written
+an article about how to start contributing to open-source and overcome the mental and
+technical barriers that come associated with it :
```
Comment (juliohm) [inline docs/src/contributing/guidelines.md:64]:
```suggestion
technical barriers that come associated with it: 
```

File: docs/src/contributing/guidelines.md [hunk 5]
```diff
@@ -55,3 +55,14 @@ that you can modify it and submit a pull request on GitHub later. Don't
...
+## I'm new to open-source. How do I start contributing?
+
+Contributing to an open-source project for the very first time can be a very daunting task.
+To make the process easier and more GitHub-beginner-friendly, the community has written
+an article about how to start contributing to open-source and overcome the mental and
+technical barriers that come associated with it: 
+
+https://invenia.github.io/blog/2021/01/29/contribute-open-source
+
+The article will also take you through the steps required to make your first contribution in detail.
```
Comment (juliohm) [inline docs/src/contributing/guidelines.md:68]:
```suggestion
The article will also take you through the steps required to make your first contribution in detail:

https://github.com/firstcontributions/first-contributions
```

File: docs/src/contributing/guidelines.md [hunk 6]
```diff
@@ -55,3 +55,16 @@ that you can modify it and submit a pull request on GitHub later. Don't
...
+## I'm new to open-source. How do I start contributing?
+
+Contributing to an open-source project for the very first time can be a very daunting task.
+To make the process easier and more GitHub-beginner-friendly, the community has written
+an article about how to start contributing to open-source and overcome the mental and
+technical barriers that come associated with it: 
+
+https://invenia.github.io/blog/2021/01/29/contribute-open-source
+
+The article will also take you through the steps required to make your first contribution in detail:
```
Comment (juliohm) [inline docs/src/contributing/guidelines.md:68]:
```suggestion
The article will also take you through the steps required to make your first contribution
in detail:
```

File: docs/src/contributing/guidelines.md (hunk 2 above)
Comment (atreyamaj) [reply docs/src/contributing/guidelines.md]: Thank you for the review! I have enabled "Allow edits from maintainers", please do let me know if it doesn't reflect that!

Committing suggested changes.

**Pull request merged**